.github/workflows/ci.yml

```diff
@@ -43,8 +43,8 @@ jobs:
           brew link --force qt@5
           # Homebrew does not link mkspecs and plugins https://github.com/Homebrew/homebrew-core/issues/93056
           export HOMEBREW_QT5_VERSION=$(brew list --versions qt@5 | rev | cut -d' ' -f1 | rev)
-          sudo ln -s /usr/local/Cellar/qt@5/$HOMEBREW_QT5_VERSION/mkspecs /usr/local/mkspecs
-          sudo ln -s /usr/local/Cellar/qt@5/$HOMEBREW_QT5_VERSION/plugins /usr/local/plugins
+          sudo ln -s /opt/homebrew/Cellar/qt@5/$HOMEBREW_QT5_VERSION/mkspecs /opt/homebrew/mkspecs
```

Comment:
Can you avoid using symbolic links? Instead, `$QMAKESPEC` may be useful. See https://doc.qt.io/qt-5/qmake-environment-reference.html

Reply:
That sounds better, but I don't have a macOS machine, so I'll need some trials on CI to get a working configuration.

Reply:
Fixed :)